core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.spec.ts

```diff
@@ -1298,6 +1298,24 @@ describe('Conversation skin component', () => {
     expect(componentInstance.mostRecentlyReachedCheckpoint).toBe('Mid');
   }));
 
+  it('should display the exploration after the the progress reminder modal' +
```

Comment:
the the -> the

Also, just from the functionality point of view, what is a "progress reminder modal"?

Reply:
Basically in my PR, for the checkpoints enabled case, what I am doing is displaying the state card only after the progress reminder modal is displayed. The progress reminder modal, as you can see in the PR video, shows the checkpoints completed by learner, and has options to restart the exploration or continue from the state left last. 

So after this modal has been shown (or it has been figured out that no checkpoints exist in the exploration), the exploration card is displayed.